administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue

```diff
@@ -9,6 +9,7 @@
             </a>
         </div>
         <media-breadcrumb></media-breadcrumb>
+        <input type="text" @input="changeSearch" placeholder="Search"/>
```

Comment:
Please use translation string

Reply:
What might be a translation string here?

Reply:
I think use <code>JText</code> @dneukirchen Right! ?

Reply:
Yes, have a look how @kasvith did it in #531.

Reply:
Yes look how i added a tranalated string